cnvlib/export.py

```diff
@@ -27,12 +27,12 @@ def label_with_gene(cnarr):
 
     if not filenames:
         return []
-    first_cnarr = read_cna(filenames[0])
+    first_cnarr = read_cna(filenames[0], None)
```

Comment:
It's not obvious what the second argument means at this point; could it be specified as a keyword argument in each case here and below?

Reply:
Yes, I can do this. Depends on whether I make the param optional in the first place (your call).